editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/adopters/util/InvalidProjectMetaDataException.java

```diff
@@ -0,0 +1,59 @@
+/******************************************************************************* 
+ * Copyright (c) 2018 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package org.fusesource.ide.projecttemplates.adopters.util;
+
+/**
+ * @author lheinema
+ */
+public class InvalidProjectMetaDataException extends Exception {
+
+	private static final long serialVersionUID = 7320130119722874052L;
+
+	/**
+	 * 
+	 */
+	public InvalidProjectMetaDataException() {
+		super();
+	}
+
+	/**
+	 * @param message
+	 * @param cause
+	 * @param enableSuppression
+	 * @param writableStackTrace
+	 */
+	public InvalidProjectMetaDataException(String message, Throwable cause, boolean enableSuppression,
```

Comment:
do we need all those constructors? I would say that only the one with the message is used so we might provide only this one, what do you think?